packages/wrangler/src/__tests__/dev.test.tsx

```diff
@@ -9,7 +9,8 @@ describe("Dev component", () => {
   beforeEach(() => (restoreConsole = patchConsole(() => {})));
   afterEach(() => restoreConsole());
 
-  it("should throw if format is service-worker and there is a public directory", () => {
+  // This test needs to be rewritten because the error now throws after a while.
```

Comment:
```suggestion
  // This test needs to be rewritten because the error now throws asynchronously
  // and the Ink framework does not yet have async testing support.
```